src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -499,8 +500,8 @@ def render(
             # filter the layers to compose to remove those who didn't have a render
         filtered_layers = [layer for layer in layers if layer['layer_id'] in output_used_layers_id]
 
-        # Prepare bg image if apply_background enabled
-        if apply_background:
+        # Prepare bg image if apply_bg_back_command
+        if apply_bg_back_command:
```

Comment:
same, this should be simply a bool, not a george script command since we won't use it in this method